test/functional/collection_tests.js

```diff
@@ -1898,4 +1898,39 @@ describe('Collection', function() {
       }
     });
   });
+
+  it('should correctly update with pipeline', {
+    metadata: {
+      requires: {
+        topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'],
```

Comment:
AFAIK we don't use the `heap` and `wiredtiger` environments anymore.

Reply:
We can handle this by remove this entire line, which will cause this test to run for every topology that we run them against.